### PR TITLE
New contributor added

### PR DESCRIPTION
**Pull request by devil-cyber**

### CODE REVIEW

File: src/data/contributors/manikant.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Manikant Kumar",
+    "skills" : ["JS", "Node.js","Python"],
+    "cause" : "Code for Community",
```
Comment (Abhishek-kumar09) [inline src/data/contributors/manikant.json:4]: Just type here the cause. **`code for`** is already the part of pre-written string. Just discover your cause of code and share it with the community. :)

You can head over to [here](https://github.com/codeforcauseorg/pledge/blob/master/hints/create_pledge.md) for insights on creating your pledge.

File: src/data/contributors/manikant.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    "image" : "Avatar3",
+    "github" : "https://github.com/devil-cyber",
+    "twitter" : "https://twitter.com/Manikan31004419",
+    "linkedin" : "https://www.linkedin.com/in/manikant-kumar-550998192/"
+}
```
Comment (Abhishek-kumar09) [inline src/data/contributors/manikant.json:9]: Please add a new Line at the end of file.

**Comment by devil-cyber** [comment]
I have changed that go through once

**Comment by Abhishek-kumar09** [comment]
Great! @devil-cyber . Also squash your commits into one because they pretty much do the same thing and welcome to code for cause :1st_place_medal:

**Pull request closed without merging**